guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -72,16 +72,16 @@ euid = __ID_of_Apache_User__
 # systemctl restart gssproxy.service
 # systemctl enable gssproxy.service
 ----
-. Configure the Apache server to use the gssproxy service:
-.. Create the `/etc/systemd/system/httpd.service` file with the following content:
+. To configure the Apache server to use the gssproxy service, create a `systemd` drop-in file and add following content in it:
```

Comment:
In the previous step, "gssproxy" is monospaced, so let's do this here too (not your fault, but if we're already changing this line…)
```suggestion
. To configure the Apache server to use the `gssproxy` service, create a `systemd` drop-in file and add following content in it:
```